### ISSUE TITLE
Error when an existing link or folder exists at the destination

### ISSUE THREAD

**Issue by djedcurtis** [ElCapitanSponge/symnode#1]
When creating a link and an existing folder or link is already in place at the destination the following occurs:

```
Error: EEXIST: file already exists, symlink 'foo' -> 'bar
    at symlinkSync (node:fs:1655:3)
    at symnode.link (node_modules\symnode\dist\symnode.js:185:34)
    at Object.<anonymous> (node_modules\symnode\dist\index.js:8:9)
    at Module._compile (node:internal/modules/cjs/loader:1103:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12)       
    at node:internal/main/run_main_module:17:47 {
  errno: -4075,
  syscall: 'symlink',
  code: 'EEXIST',
  path: '../iris/clients',
  dest: 'client'
}
```

**Comment by ElCapitanSponge** [comment]
@djedcurtis work will begin shortly on this as it is a critical issue

**Comment by ElCapitanSponge** [comment]
Resolved in #2

**Comment by ElCapitanSponge** [comment]
Closing as resolved
